### DISCUSSION TITLE
Olric nodes communication

### DISCUSSION THREAD

**Discussion by Ritesh-dotcom** [olric-data/olric#104] [General]
Hi!
Nice project !
I was running the sample single node example, and it executed nicely.
However, I unable to do this with multiple nodes.
Can anyone explain me how to create a multiple nodes Olric cluster?

**Comment by jrobin42** [comment]
Hi ! 
Same here, I let you know if I succeed to do this

**Comment by buraksezer** [comment]
Hi @Ritesh-dotcom and @jrobin42 

I'm so sorry for the latency. Olric has two different deployment modes: embedded-server and client-server. I assume that you are trying to create a cluster in embedded-server mode. You can see same usage samples in the following documents:

- https://github.com/buraksezer/olric-consul-plugin
- https://github.com/buraksezer/olric-cloud-plugin

You can manage the peer list manually for experimenting but it's a good idea to use a plugin for service discovery. 

If you any further questions, please don't hesitate to contact me.
